tide/core/scene/ContentWindow.h

```diff
@@ -144,7 +144,7 @@ class ContentWindow : public Rectangle
     /** @return the unique identifier for this window. */
     const QUuid& getID() const;
 
-    /** Is the window a panel */
+    /** Is the window a panel. */
```

Comment:
@return true if window is a panel

Maybe describe even shortly what a panel is/differentiates from a 'normal' window. I don't know now actually :)